src/implementations/vanilla/src/basics/text-link/text-link.html

```diff
@@ -0,0 +1 @@
+<a class='hig__text-link hig__text-link__{{ type }}'>{{ text }}</a>
```

Comment:
should "href" also be interpolated in here if it is a default?

Reply:
Actually isn't a default, or at least the default is null.  Some links with onclick events won't have href set

Reply:
ok cool.